src/modules/common/models/team.model.ts

```diff
@@ -77,6 +78,10 @@ export class Team {
   @IsString()
   @IsOptional()
   updatedBy?: string;
+
+  @IsBoolean()
+  @IsOptional()
+  isNewInvite?: boolean;
```

Comment:
Remove it from Team Model. Team Dto alone should be fine.

Reply:
resolved